### PR TITLE
add anyio

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes/anyio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - python >=3.6
```
Comment (marcelotrevisani) [inline recipes/anyio/meta.yaml]: It seems that there are some dependencies for this project
```suggestion
    - python >=3.6
    - sniffio
    - curio
    - trio
    
```

File: recipes/anyio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "0f47b5ea774cc9d403935c617ff27c7673f87cda0688cd1aab3137ac86283a68"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/anyio/meta.yaml]: Should it be?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```
https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---vv

Comment (CJ-Wright) [reply recipes/anyio/meta.yaml]: It seems that skeleton needs an update, maybe?

Comment (marcelotrevisani) [reply recipes/anyio/meta.yaml]: I will do it now, otherwise, I will forget about it!
Thanks! :D

**Comment by CJ-Wright** [comment]
Thank you very much for the review!

**Pull request merged**